ironfish/src/primitives/rawTransaction.test.ts

```diff
@@ -170,7 +178,9 @@ describe('RawTransaction', () => {
     })
 
     expect(() => raw.post(account.spendingKey)).toThrow(
-      'Cannot post transaction. Burn value exceededs maximum',
+      `Cannot post transaction. Burn value ${CurrencyUtils.renderIron(
+        burn.value,
+      )} exceededs maximum ${CurrencyUtils.renderIron(MAX_MINT_OR_BURN_VALUE)}`,
```

Comment:
Don't change these errors in the test, you can do partial checking of an error using `toThrowError`

Reply:
Try to match error with regex in the latest commit.